src/components/Modals/FiatRamps/views/FiatForm.tsx

```diff
@@ -46,18 +51,27 @@ export const FiatForm: React.FC<FiatFormProps> = ({
   const { data: ramps } = useGetFiatRampsQuery()
   const { assetSearch } = useModal()
 
+  const buyAssets = useMemo(() => {
+    const buyAssetIds = ramps?.buyAssetIds ?? []
+    return buyAssetIds.map(assetId => sortedAssetsById.get(assetId)!)
```

Comment:
careful with non-null assertions, this is causing fiat page to crash as fiat ramps often support more assets than we do

Reply:
[`77cbf8e` (#4292)](https://github.com/shapeshift/web/pull/4292/commits/77cbf8ebf783653ce2c017af926639e2c474ccfb)